docs/user-guide/api/index.rst

```diff
@@ -0,0 +1,21 @@
+.. _data-curator-api:
+
+======================================
+API Reference
+======================================
+
+.. toctree::
+   :maxdepth: 4
+   :titlesonly:
```

Comment:
What is this for?

Reply:
It's so that all the options show up in the sidebar navigation menu.